tests/foreman/maintain/test_service.py

```diff
@@ -124,7 +124,8 @@ def test_positive_service_stop_start(sat_maintain):
     assert result.status == 0
 
 
-@pytest.mark.e2e
+@pytest.mark.stream
```

Comment:
stream mark should be used for stream specific features, and not for tests in zstream branches